crates/sui-bridge/src/crypto.rs

```diff
@@ -63,27 +66,41 @@ impl BridgeAuthoritySignInfo {
     }
 }
 
-pub fn verify_signed_bridge_event(
-    e: SignedBridgeAction,
+/// Verifies a SignedBridgeAction (response from bridge authority to bridge client)
+/// represents the right BridgeAction, and is signed by the right authority.
+pub fn verify_signed_bridge_action(
+    expected_action: &BridgeAction,
+    signed_action: SignedBridgeAction,
+    expected_signer: &BridgeAuthorityPublicKeyBytes,
```

Comment:
why is this needed, isn't the committee enough?

Reply:
this function is called in BridgeClient::request_sign_bridge_action, where client requests a server to sign an action. Passing the `expected_signer` to match `&sig.authority_pub_key_bytes()` is to make sure the server returns its own signature (e.g. not another authority's, which may also be in the committee)